docs/demos.md

```diff
@@ -0,0 +1,29 @@
+# AMT Demos 
+
+These documentations give an overview of how the demos work and explain some mechanism in these demos. (including some functions we use in the demo)
```

Comment:
"The documents listed below give"
"explain some mechanisms in these"

You can remove "(including some functions we use in the demo)" I think it's implied already.